pkgs/development/python-modules/django-webpush/default.nix

```diff
@@ -13,8 +12,6 @@ buildPythonPackage rec {
   version = "0.3.4";
   pyproject = true;
 
-  disabled = pythonOlder "3.7";
-
```

Comment:
Could you please leave this line?
This parameter is useful for those using the old Python.